api/grpc/server_integ_test.go

```diff
@@ -132,8 +132,9 @@ func Test_Integ_Role(t *testing.T) {
 	var alicePeer, bobPeer *pb.Peer
 	var chID string
 	prng := rand.New(rand.NewSource(1729))
-	aliceCfgFile := sessiontest.NewConfigFile(t, sessiontest.NewConfig(t, prng))
-	bobCfgFile := sessiontest.NewConfigFile(t, sessiontest.NewConfig(t, prng))
+	aliceCfg, bobCfg := sessiontest.NewConfig(t, prng), sessiontest.NewConfig(t, prng)
```

Comment:
Why this is changed so?

Reply:
I was changing somethings else and forgotten to undo it I guess. Reverting to the older state.
```suggestion
	aliceCfgFile := sessiontest.NewConfigFile(t, sessiontest.NewConfig(t, prng))
	bobCfgFile := sessiontest.NewConfigFile(t, sessiontest.NewConfig(t, prng))
```